NeosModLoader/NeosVersionReset.cs

```diff
@@ -68,8 +71,11 @@ internal static void Initialize()
 				})
 				.ToArray();
 
-			string actualPlugins = plugins.Keys.Join(delimiter: ", ");
-			Logger.DebugFuncInternal(() => $"Found {plugins.Count} actual plugins: {actualPlugins}");
+			if (Logger.IsDebugEnabled())
+			{
+				string actualPlugins = plugins.Keys.Join(delimiter: ", ");
+				Logger.DebugInternal($"Found {plugins.Count} actual plugins: {actualPlugins}");
+			}
```

Comment:
Huumh, couldn't the debug methods just take in a closure that returns the debug string, and only gets called if the debug level is enabled...? Would be pretty clean to move even the normal things to that format, and would avoid this kind of if conditions checking x'P  though it would be a bit over-engineered x'P

Reply:
Actually... that's a good idea. Silly me for not seeing it.